src/plugins/data/server/search/strategies/ese_search/ese_search_strategy.ts

```diff
@@ -53,7 +53,6 @@ export const enhancedEsSearchStrategyProvider = (
       request.params?.keep_alive ?? getDefaultAsyncGetParams(searchConfig, options).keep_alive;
 
     const { body, headers } = await client.asyncSearch.status(
-      // @ts-expect-error keep_alive was recently added and the types haven't been updated yet
```

Comment:
so the `keep_alive` param was added or removed? because seems to contradict the other comments

Reply:
It's now available in in the request `asyncSearch.status`, but removed from others :thisisfine: